design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,109 @@
+# Implementing Authenticator Status Checks
+The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
+operators can make a RESTful request using the Conjur API and receive a response identifying if configuration was successful.
+If not, the operators receive the proper error identifying the step of failure. 
+The first authenticator to utilize this feature is Authn OpenID Connect. 
+
+_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status check functionality to other authenticators.
+For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
+checks are run followed by those specific to the authenticator.  
+
+General checks include:
+
+**NOTE:** Steps that require developer intervention to implement the status check are italicized below
+1. The requesting user has access to the authenticator status webservice
+    1. See the security section [here](https://app.zenhub.com/workspaces/appliance-development-5c9cf04bfa1c204b63eddcc6/issues/cyberark/conjur/1062) for more info about this webservice 
```

Comment:
Please don't link to zenhub as it requires authorization. link to GH: https://github.com/cyberark/conjur/issues/1062

Reply:
you can get that link from the zanhub issue

Reply:
zan zan zan hub